server/src/com/thoughtworks/go/server/service/PipelineConfigService.java

```diff
@@ -184,7 +184,7 @@ public void onPipelineConfigChange(PipelineConfig pipelineConfig, String group)
 
     @Override
     public void onConfigChange(CruiseConfig newCruiseConfig) {
-        PipelineConfigurationCache.getInstance().onConfigChange(goConfigService.getConfigForEditing());
+        PipelineConfigurationCache.getInstance().onConfigChange(goConfigService.cruiseConfig());
```

Comment:
@jyotisingh — `getConfigForEditing` has been deprecated, but is still being used in a lot of places. Should we be replacing all occurrences to avoid such bugs?